pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,26 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+RUN apt-get -y --force-yes update \
+    && apt-get install -y --fix-missing \
+    wget \
+    cpio \
+    libpcre3-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libsm6 \
+    libxt6 \
+    libglu1-mesa \
+    && pip install xmldiff
```

Comment:
FWIW, I would disagree with @bam241 on this point.  Since `apt` and `pip` can fail in different ways/for different causes, I would rather have the `apt` line record a hash in the build process and then have the `pip` line act separately for efficiency

Reply:
And we `pip` in an entirely different place in the `pymoab` scripts anyway

Reply:
I guess @bam241 and I both claimed efficiency....

I'm talking about an efficiency for rebuilding in which changes to the `pip` line don't require repeating the `apt` line, which can sometimes be long-running

Reply:
my feeling is those apt/pip line are not the sensitive part of the build process, so they could be merged into a single one.
But I also hear your point.

I don't have strong feelings there... (Sorry @kkiesling ) ... and I don't know what are the best practices in that case

Reply:
I agree with @gonuke in this case, because I made a lot of changes to my `pip` lines when I was building my docker image and it was helpful that it didn't have to completely redo the `apt` line each time, which was the lengthier process.

Reply:
I am also team two lines for the same reasons listed here by @gonuke and @piperlincoln (sorry @bam241). I am changing it back.